pkg/api/testapi/testapi.go

```diff
@@ -257,6 +258,15 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
+	if _, ok := Groups[kubeadm.GroupName]; !ok {
```

Comment:
This whole package should not exist, we should reuse the manager in apimachinery/registered. It's a big change though. @mml This is one of the refactoring changes I want to make.